tap_github/schemas/pull_requests.json

```diff
@@ -8,6 +8,1274 @@
     "id": {
       "type": ["null", "string"]
     },
+    "node_id": {
+      "type": ["null", "string"]
+    },
+    "statuses_url": {
+      "type": ["null", "string"]
+    },
+    "draft": {
+      "type": ["null", "boolean"]
+    },
+    "requested_reviewers": {
```

Comment:
Same field in review_comments has string type, it is expected, right?

Reply:
This was already present in the schema, and not present in the API doc, however, we have not removed it.